feature_engine/outliers/trimmer.py

```diff
@@ -89,6 +89,31 @@ class OutlierTrimmer(WinsorizerBase):
     transform:
         Remove outliers.
 
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> from feature_engine.outliers import OutlierTrimmer
+    >>> np.random.seed(42)
+    >>> X = pd.DataFrame(dict(x = np.random.exponential(size = 100)))
+    >>> ot = OutlierTrimmer(capping_method='gaussian', tail='right', fold=3)
```

Comment:
If we are using gaussian, I think it would be better to use np.random.normal instead of exponential.

These snippets are meant for the readers to be able to copy and paste into their code. We should just demo how to set the transformer and then how to fit it and transform data, showing the resulting dataframe.

I'd remove all the intermediate prints. For more details we have a user guide that users can read.

I'd also use a simpler data for the demo, with 10 values, instead of a random extraction of 100 values. Could we update the example ?